.github/workflows/ci_cd.yml

```diff
@@ -181,12 +182,6 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
-      - id: paths-filter
-        name: Conditionally skip job
-        uses: dorny/paths-filter@v2
-        with:
-          filters: .github/filters.yml
-
```

Comment:
Why is this removed?

Reply:
This step was not used or referenced anywhere. All jobs use the `get-changes` job to determine what has changed.